optashift-employee-rostering-shared/src/main/resources/META-INF/persistence.xml

```diff
@@ -3,15 +3,17 @@
              version="2.1" xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/persistence http://xmlns.jcp.org/xml/ns/persistence/persistence_2_1.xsd">
   <persistence-unit name="optashift-employee-rostering-persistence-unit" transaction-type="JTA">
     <description>Hibernate Persistence Unit</description>
-    <jta-data-source>java:jboss/datasources/ExampleDS</jta-data-source>
+    <jta-data-source>${optashift.rostering.persistence.datasource:java:jboss/datasources/ExampleDS}</jta-data-source>
```

Comment:
Causing a build failure for me; need to set the default value; should be ${optashift.rostering.persistence.datasource} and you set <optashift.rostering.persistence.datasource>java:jboss/datasources/ExampleDS</optashift.rostering.persistence.datasource> to properties

Reply:
Well, we need some way to tell wildfly in "wildfly:start wildfly:deploy" to inject those variables (when attempted to run, wildfly reports: "failed to launch service {...${optashift.rostering.persistence.datasource:java:jboss/datasources/ExampleDS}")

Reply:
(which should have expanded to "java:jboss/datasources/ExampleDS" (in other words, the wildfly plugin does not expand/replace/do whatever to the variable)

Reply:
I tested on EAP 7, used this setting in standalone-openshift.xml, it should work in eap6 and wildfly 10 as well. change this from false to true, it allows parameter replacement.
<spec-descriptor-property-replacement>false</spec-descriptor-property-replacement>

Reply:
BTW, this is where I set the spec-descriptor-property-replacement value to true 

https://github.com/calvinzhuca/jboss-kie-modules/blob/master/os.optashift/added/openshift-launch.sh